src/lazy/sequence.rs

```diff
@@ -282,6 +297,15 @@ impl<'a, 'top, 'data: 'top, D: Decoder> IntoIterator for &'a LazySExp<'top, D> {
     }
 }
 
+impl<'top, 'data: 'top, D: Decoder> IntoIterator for LazySExp<'top, D> {
```

Comment:
🗺️ Now that `LazyList`, `LazySExp`, and `LazyStruct` are `Copy`, it makes sense to allow them to be iterated over without having to borrow them first.